src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -343,14 +343,6 @@ export class CalcitePopover {
   //
   // --------------------------------------------------------------------------
 
-  renderImage(): VNode {
```

Comment:
how come this was removed? This would be a breaking change.

Reply:
The doc for the slot would have to be removed too.

```
/**
 * @slot image - A slot for adding an image. The image will appear above the other slot content.
 */
```

The commit message would need to change to specify the breaking change.

Reply:
I can update the doc and the PR to reflect the breaking change.
This image slot was added when the component was created and was never used.
Checked with @julio8a on this and got the go-ahead to remove it.